### DISCUSSION TITLE
Using existing Haproxy

### DISCUSSION THREAD

**Discussion by alitvak69** [roxy-wi/roxy-wi#309] [Q&A] [answered]
Happy New Year,
I wanted to add existing haproxy already running in a docker container.  Wanted to have gui for monitoring and api access.  Is it even possible?  Documentation is not clear on that.  Can you please let me know?

**Comment by Aidaho12** [comment, marked as answer]
Happy New year!

Just add a server and mark it as a docker, and check settings, if you have custom

**Reply by alitvak69** [reply]
Thank you for reply,

Does docker has to have SSH running?  We usually don't allow that.

**Comment by Aidaho12** [comment]
Docker doesn't, but server where is docker has to

**Reply by alitvak69** [reply]
Do you work only with docker or also with other containers like podman ?

**Comment by Aidaho12** [comment]
Only docker
